exercises/hamming/hamming_test.py

```diff
@@ -3,10 +3,13 @@
 import hamming
 
 
-# test cases adapted from `x-common//canonical-data.json` @ version: 1.0.0
+# Tests adapted from `problem-specifications//canonical-data.json` @ v2.0.1
 
 class HammingTest(unittest.TestCase):
-
+    
+    def test_identical_strands(self):
```

Comment:
This function shares it's name with the one below, so it's just getting redefined without use. You need to change the name of one of these functions.